### ISSUE TITLE
Compatibility problem

### ISSUE THREAD

**Issue by Jmadih** [joonhocho/react-native-linkedin-sdk#7]
I'm new in react native apps and i'm trying to run an app that use your sdk, but something is not compatible... 
I got the error =>
* What went wrong:
A problem occurred configuring project ':app'.
> Could not resolve all dependencies for configuration ':app:_debugApk'.
   > A problem occurred configuring project ':react-native-linkedin-sdk'.
      > The SDK Build Tools revision (23.0.1) is too low for project ':react-native-linkedin-sdk'. Minimum required is 25.0.0

Something install the 23.0.1 sdk i don't know why !
here is my package.json
  "dependencies": {
    "axios": "^0.15.3",
    "native-base": "^2.1.1",
    "react": "^15.5.3",
    "react-native": "^0.43.2",
    "react-native-linkedin-sdk": "0.0.4",
    "react-native-vector-icons": "^4.0.1",
    "react-navigation": "^1.0.0-beta.7"
  },

**Comment by Asha-Nellaiappan** [comment]
Change it into react-native-linkedin-sdk/android/build.gradle

```
android {
    compileSdkVersion 25
    buildToolsVersion "25.0.0"
    defaultConfig {
        minSdkVersion 16
        targetSdkVersion 25   
    }
}
```

As far as i know this issue may be occurred because of your react native android app has higher build version. Please check your android/app/build.gradle file build tool version. 

Change the react-native-linkedin-sdk/android/build.gradle- build tool version equal to your android app build tool version.

After change rebuild/run your react native project.

**Comment by joonhocho** [comment]
@Asha-Nellaiappan @Jmadih Yeah, I've just looked into this issue, and it depends on your project settings. I am not sure if it's wise to bump those versions for this library, since it may cause errors for those using lower build tools.
